checker/build.xml

```diff
@@ -510,6 +520,15 @@
     <!-- Used to guarantee that targets executed from other ant scripts do not cause unnecessary rebuilds -->
     <target name="all-tests-nojar" depends="build-tests"
             description="Run tests for all checkers, WITHOUT building anything">
+
```

Comment:
How about adding a comment to the two duplicates to make sure anyone who edits one also edits the other?
And adding an issue that we should fix the duplication.